en/components/toggle.md

```diff
@@ -83,109 +63,77 @@ export class Class {
         closeOnOutsideClick: false,
-        modal: false,
-        positionStrategy: new ConnectedPositioningStrategy(this._positionSettings),
-        scrollStrategy: new CloseScrollStrategy()
+        positionStrategy: new ConnectedPositioningStrategy(this._positionSettings)
     };
+
     public toggle() {
         this._overlaySettings.positionStrategy.settings.target = this.igxButton.nativeElement;
         this.igxToggle.toggle(this._overlaySettings);
     }
 ```
 
+This is how our toggle should look like now:
 
 <div class="sample-container loading" style="height: 370px">
-    <iframe id="toggle-iframe-1" data-src='{environment:demosBaseUrl}/interactions/toggle' width="100%" height="100%" seamless="" frameBorder="0" class="lazyload"></iframe>
+    <iframe id="toggle-iframe" src='{environment:demosBaseUrl}/interactions/toggle' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
 </div>
 <div>
-    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="toggle-iframe-1" data-demos-base-url="{environment:demosBaseUrl}">                       view on stackblitz
+    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="toggle-iframe" data-demos-base-url="{environment:demosBaseUrl}">                       view on stackblitz
     </button>
 </div>
 
-***Note***: By defaut [`closeOnOutsideClick`]({environment:angularApiUrl}/interfaces/overlaysettings.html#closeonoutsideclick) is set to `false`. If set to `true` there are some limitations in the behavior. When element is shown clicking on toggle button will force close on outside click and toggle method. Close on outside click will start close animation. Then toggle method, with close animation started, will call open. Those two events neutralize each other and nothing happens. In order to achieve the desired behavior there a couple of options listed below: 
-- use [`IgxToggleActionDirective`]({environment:angularApiUrl}/classes/igxtoggleactiondirective.html) as described below;
-- set [`closeOnOutsideClick`]({environment:angularApiUrl}/interfaces/overlaysettings.html#closeonoutsideclick) to `false` as in the sample above
-
-### Automatic toggle actions
-
-In order to prevent the invocation of these methods there is a directive which has `onClick` handler and changes the state to the toggle we are referred to. So let's dive in. If we would like to take advantage of this functionality we will have to use [**IgxToggleActionDirective**]({environment:angularApiUrl}/classes/igxtoggleactiondirective.html) which is coming from the same **IgxToggleModule**.
+#### Automatic toggle actions
 
-```typescript
-// app.module.ts
-
-...
-import { IgxToggleModule, IgxButtonModule } from 'igniteui-angular'
+In order to avoid using the `open` and `close` methods, we can operate with a directive, which has `onClick` handler, and changes the state of the toggle we are referring to automatically.
```

Comment:
```suggestion
In order to avoid using the `open` and `close` methods, we can use a directive, which has an `onClick` handler and changes the state of the toggle we are referring to automatically.
```